<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Recette">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Description">
     <attribute name="title">
      <string>Explications</string>
     </attribute>
     <widget class="QListView" name="listeDescription">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>561</width>
        <height>411</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="autoScroll">
       <bool>true</bool>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="resizeMode">
       <enum>QListView::Adjust</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="itemAlignment">
       <set>Qt::AlignLeading</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Ingredients">
     <attribute name="title">
      <string>Ingr√©dients</string>
     </attribute>
     <widget class="QListView" name="listeIngredient">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>561</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Etapes">
     <attribute name="title">
      <string>Etapes</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QListView" name="VueTemps">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>520</y>
      <width>801</width>
      <height>41</height>
     </rect>
    </property>
    <property name="flow">
     <enum>QListView::LeftToRight</enum>
    </property>
   </widget>
   <widget class="QListView" name="VueURL">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>500</y>
      <width>451</width>
      <height>21</height>
     </rect>
    </property>
    <property name="flow">
     <enum>QListView::LeftToRight</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
